storage/innobase/xtrabackup/src/xb_dict.cc

```diff
@@ -0,0 +1,49 @@
+/******************************************************
+Copyright (c) 2023 Percona LLC and/or its affiliates.
+
+interface containing map/set required for PXB
+
+This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation; version 2 of the License.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program; if not, write to the Free Software
+Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA
+
+*******************************************************/
+#include "xb_dict.h"
+#include <memory>
+#include "backup_mysql.h"
+
+std::shared_ptr<xb_dd_tablespaces> xb_build_space_id_set(MYSQL *connection) {
```

Comment:
lets use namespace xb?
then function name as build_dd_spaces_set()?

Reply:
moved to namespace